src/incubator/toast/helpers/useToastPresets.ts

```diff
@@ -0,0 +1,75 @@
+import 'react';
+import {Colors} from 'style';
+import {ToastProps, ToastPreset} from '../types';
+
+const checkMarkIcon = require('../assets/checkmarkFlat.png');
+const exclamationIcon = require('../assets/exclamationFill.png');
+const infoIcon = require('../assets/info.png');
+const redCloudIcon = require('../assets/redCloud.png');
+
+export default ({preset, icon, iconColor, message}: Pick<ToastProps, 'preset' | 'icon' | 'message' | 'iconColor'>) => {
```

Comment:
Why do you use `Pick<ToastProps, ...>` when you extract the props?

Reply:
Because I want the hook to have an accurate typecheck. 
Instead of passing all props, I declare only the ones relevant to this hook